tools/build_pytorch_libs.py

```diff
@@ -218,6 +218,11 @@ def run_cmake(version,
     if mkldnn_threading:
         cmake_defines(cmake_args, MKLDNN_THREADING=mkldnn_threading)
 
+    parallel_backend = os.getenv('PARALLEL_BACKEND')
+    if not parallel_backend:
+        parallel_backend = 'OPENMP'
```

Comment:
don't you already have default in cmake? also, if it's not the - let's not add additional cmake_defines